### PR TITLE
fix script

### PR DESCRIPTION
**Pull request by krivtsov**

### CODE REVIEW

**Review by andymunro — APPROVED** [review]

**Review by andymunro — APPROVED** [review]

**Review by ahus1 — APPROVED** [review]
Thank you for fixing this script!

**Comment by ahus1** [comment]
@krivtsov - there is no need to update the PR. It is now waiting for a maintainer to be merged. 

At the maintainers: please squash all commits in the PR. Thanks!

**Review by ahus1 — APPROVED** [review]

**Review by stianst — APPROVED** [review]

**Pull request merged**
